docs/Basic-Guide.md

```diff
@@ -84,30 +93,26 @@ More information and documentation is provided in the
 
 ## Training the Brain with Reinforcement Learning
 
-### Adding a Brain to the training session
+### Setting up the enviornment for training
 
 To set up the environment for training, you will need to specify which agents are contributing
 to the training and which Brain is being trained. You can only perform training with
 a `Learning Brain`.
 
-1. Assign the **3DBallLearning** Brain to the agents you would like to train.  
-   __Note:__ You can assign the same Brain to multiple agents at once : To do so, you can
-   use the prefab system. When an agent is created from a prefab, modifying the prefab 
-   will modify the agent as well. If the agent does not synchronize with the prefab, you
-   can hit the Revert button on top of the Inspector.
-   Alternatively, you can select multiple agents in the scene and modify their `Brain`
-   property all at once.
-2. Select the **Ball3DAcademy** GameObject and make sure the **3DBallLearning** Brain
-   is in the Broadcast Hub. In order to train, you need to toggle the
-   `Control` checkbox.
-   
-__Note:__ Assigning a Brain to an agent (dragging a Brain into the `Brain` property of 
+1. Each platform agent needs an assigned `Learning Brain`.  In this example, each platform agent was created using a prefab.  To update all of the brains in each platform agent at once, you only need to update the platform agent prefab.  In the **Project** window, go to the `Assets/ML-Agents/Examples/3DBall/Prefabs` folder. Expand `Game` and click on the `Platform` prefab.  You should see the `Platform` prefab in the **Inspector** window.  In the **Project** window, drag the **3DBallLearning** Brain located in  `Assets/ML-Agents/Examples/3DBall/Brains` into the `Brain` property under `Ball 3D Agent (Script)` component in the **Inspector** window.  
+
+   **Note**: The Unity prefab system will modify all instances of the agent properties in your scene.  If the agent does not synchronize automatically with the prefab, you can hit the Revert button in the top of the **Inspector** window.
+
+2. In the **Hierarchy** window, select `Ball3DAcademy`.
+3. In the **Project** window, go to `Assets/ML-Agents/Examples/3DBall/Brains` folder and drag the **3DBallLearning** Brain to the `Brains` property under `Braodcast Hub` in the `Ball3DAcademy` object in the **Inspector** window.  In order to train, make the `Control` checkbox is selected.
```

Comment:
is selected -> selected

Reply:
Maybe it was "make _sure_ the `Control` checkbox is selected?

Reply:
should be make sure